packages/loading-indicator/src/BlobLoader/BlobLoader.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import { StoryMeta } from '@leafygreen-ui/lib';
+
+import BlobLoader from './BlobLoader';
+
+export default StoryMeta({
+  title: 'Components/LoadingIndicator/BlobLoader',
+  component: BlobLoader,
+  argTypes: {
+    description: { control: 'text' },
```

Comment:
this should respond to baseFontSize

Reply:
It does respond to baseFontSize - it just doesn't need to be explicitly added to this Storybook file since it's added by default globally